ls1intum/Artemis
# Fullscreen mode on modelling-submission.component

### Checklist

### Motivation and Context


This change implements the feature request for #470 
It provides the possibility to toggle full screen mode for modelling exercises, so that students can present their modelling solutions. It also includes the feedback of the tutor on the side

### Description

added a fullscreen module in shared code which can provide this functionality to all wrapped children. I also included it in the modelling-submission.component

### Steps for Testing


[Create a modelling exercise]
1. Log in to Artemis
2. Navigate to Course Management
3. Select "Exercises" of your test course
4. Create a new Modelling Exercise

[Create a manual graded submission]
1. Go to Course Overview
2. Select your test course
3. Start Modelling Exercise
4. Do Modelling Exercise
5. Submit Modelling Exercise
6. Go to Course Management 
7. Select Tutor-Dashboard of your Exercise
8. Participate in Assessment
9. Assess the previous submitted modelling exercise

[Navigate to fullscreen]
1. Navigate to Course Overview
2. Select your test course
3. Go to you modelling exercise
4. Click View Results
5. On the bottom right of the assessment section, you will find the toggle Fullscreen button
6. See if fullscreen mode works

### Screenshots


![fullscreen_modelling_editor](https://user-images.githubusercontent.com/22955066/80315889-cd0d5c00-87fa-11ea-9e56-c9b7229ce88b.gif)
